students/WesWang/lesson5/mailroom_part3.py

```diff
@@ -0,0 +1,106 @@
+#=========================================================================================
+#Name: Mailroom
+#Author: Wesley Wang
+#Date: 4/26/2018
+#=========================================================================================
+
+"""
+Mailroom Part3
+"""
+import sys
+from datetime import date
+
+
+donors = {"Donor A":[100000, 40000]
+        , "Donor B":[1000]
+        , "Donor C":[1000,1000,1000]
+        , "Donor D":[5000,10000,5000]}
+
+
+def thank_letter(donor, amount):
+    letter = "-"*80 + f"\nDear {donor},\n\n\tWe have received you donation of ${amount:,.2f}.\n\n\tWe greatly appreciate your generous contribution.\n\n\tThis donation is meaningful and crucial to our organization's misson.\n\n\nSincerely,\n\nWesley Wang\n" + "-"*80
```

Comment:
Some people will get annoyed that this line is so long, so in the real world it's usually best to keep lines shorter than 79 chars to keep things pep8 compliant.